pages/linux/secret-tool.md

```diff
@@ -0,0 +1,21 @@
+# secret-tool
+
+> Store and retrieve passwords, part of the `libsecret` package.
+> Communicates with `Secret Service` implementation such as `gnome-keyring`.
+> For more information <https://wiki.gnome.org/Projects/Libsecret>.
```

Comment:
```suggestion
> More information: <https://gnome.pages.gitlab.gnome.org/libsecret/>.
```
Since this GNOME Wiki page will be archived soon, I would suggest directly linking to the docs instead.